### PR TITLE
Add LumaLabs Video Generation

### PR DESCRIPTION
**Pull request by Ayush0054**
## Description

**Please include:**
- added lumalabs video generator toolkit
- text to video, image to video functionality
- added lumalabs_tool in cookbook/tools

Fixes # (issue)

## Type of change

Please check the options that are relevant:

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Model update
- [ ] Infrastructure change

## Checklist

- [x] My code follows Phidata's style guidelines and best practices
- [x] I have performed a self-review of my code
- [ ] I have added docstrings and comments for complex logic
- [x] My changes generate no new warnings or errors
- [x] I have added cookbook examples for my new addition (if needed)
- [ ] I have updated requirements.txt/pyproject.toml (if needed)
- [x] I have verified my changes in a clean environment

## Additional Notes

Include any deployment notes, performance implications, or other relevant information:

### CODE REVIEW

File: phi/tools/lumalab.py [hunk 1]
```diff
@@ -0,0 +1,174 @@
...
+
+        self.client = LumaAI(auth_token=self.api_key)
+        self.register(self.generate_video)
+        self.register(self.image_to_video)
+
+    def image_to_video(
+        self,
+        agent: Agent,
+        prompt: str,
+        image_url: str,
```
Comment (dirkbrnd) [inline phi/tools/lumalab.py:51]:
```suggestion
        start_image_url: str,
```

File: phi/tools/lumalab.py [hunk 2]
```diff
@@ -0,0 +1,103 @@
...
+from phi.tools import Toolkit
+from phi.utils.log import logger
+
+try:
+    from lumaai import LumaAI
+except ImportError:
+    raise ImportError("`lumaai` not installed. Please install using `pip install lumaai`")
+
+
+class LumaLab(Toolkit):
```
[the remark this answers is not in the document]
Comment (Ayush0054) [reply phi/tools/lumalab.py:25]: done

File: phi/tools/lumalab.py (hunk 1 above)
Comment (Ayush0054) [reply phi/tools/lumalab.py:51]: done

File: phi/tools/lumalab.py [hunk 3]
```diff
@@ -0,0 +1,103 @@
...
+        Args:
+            prompt (str): A text description of the desired video.
+            loop (bool, optional): Whether the video should loop. Defaults to False.
+            aspect_ratio (str, optional): Aspect ratio of the video. Defaults to "16:9".
+            keyframes (Dict[str, Any], optional): Keyframe configuration for image-to-video or video extension.
+
+        Returns:
+            str: A message indicating if the video has been generated successfully or an error message.
+        """
+        if not self.api_key:
```
[the remark this answers is not in the document]
Comment (Ayush0054) [reply phi/tools/lumalab.py:69]: done

**Review by dirkbrnd — APPROVED** [review]

**Pull request merged**